### DISCUSSION TITLE
0.5.7

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1390] [Announcements]
- Discord RPC support!
- In-app update checker
- Player text alignment customization
- Exclude downloaded files in auto backup
- Fix random crash
- Other fixes and enhancements

<details>
<summary>Screenshot</summary>

Player text align sided

<img src="https://github.com/user-attachments/assets/2f8cc7ae-3b14-477b-a482-d22965b0bf34" width=200 />

</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.7'>0.5.7</a>.</em>

**Comment by Anonuser-code** [comment]
Thanks for fixes dev. U best. ♥️

**Comment by paula879** [comment]
The equalizer closes after each song

**Reply by z-huang** [reply]
Can't reproduce.

**Comment by chamudha** [comment]
Pure black is notworking correctly .![IMG_20240817_063005.jpg](https://github.com/user-attachments/assets/cd5c84a4-4b2b-4e56-9e9c-d16f675cd4e1)

**Reply by z-huang** [reply]
Material 3 updated some specs. It's not a bug.

**Comment by CesarF08** [comment]
The update is perfect. If I had to point out a "flaw," it would be that the lyrics are one line ahead of the song. Other than that, everything is perfect; errors have been corrected properly.

**Reply by HehehBoiii78** [reply]
Actually it's not the devs fault. Lyrics, especially timed lyrics are provided by KuGou, and I've also noticed this thing. They definitely don't time the lyrics carefully.

**Comment by Roxo7777** [comment]
A fork of InnerTune (OpenTune), has the option of from 320kpbs quality, and actually some songs when I enter "Details" of the song, shows the quality 320kbps (although few songs came with this quality)
![Screenshot_20240817_050055_OpenTune.jpg](https://github.com/user-attachments/assets/167df015-5259-4039-8be3-bcec8a33515e)

**Reply by z-huang** [reply]
Can you put the link of OpenTune?

**Reply by Roxo7777** [reply]
> Can you put the link of OpenTune?

https://innertunne.netlify.app/en

**Reply by z-huang** [reply]
You may want to see this issue https://github.com/Arturo254/InnerTune/issues/77, which the author didn't explain clear about why he can get 320kbps even though people only get 168kbps. In fact, the quality we get is determined by YouTube. I've never seen YouTube giving us 320kbps. Oh, if you're logged in with a premium account, it may be possible.

**Reply by Roxo7777** [reply]
> You may want to see this issue https://github.com/Arturo254/InnerTune/issues/77, which the author didn't explain clear about why he can get 320kbps even though people only get 168kbps. In fact, the quality we get is determined by YouTube. I've never seen YouTube giving us 320kbps. Oh, if you're logged in with a premium account, it may be possible.

So/oh (for "ata" in portuguese), now it makes sense, thank you =)

**Reply by hewiofhiowa** [reply]
YouTube doesn't do 320kbps, the highest I've seen is 256 kbps opus, which I have only found for one video. For most songs the highest quality available is 128kbps opus

**Reply by Roxo7777** [reply]
> YouTube doesn't do 320kbps, the highest I've seen is 256 kbps opus, which I have only found for one video. For most songs the highest quality available is 128kbps opus

In InnerTune, no, sometimes it hits 156kpbs, 180kpbs... It varies with availability.

**Comment by BocchiTheTogenashiTogeari** [comment]
Just discovered this earlier this morning and man is this amazing, thank you for your amazing work but I do have one problem that I've been encountering and that's the app crashing whenever I press "view artist" On tn-shi's songs/videos

**Reply by Roxo7777** [reply]
Me too, I already reported in Issues

**Reply by Anonuser-code** [reply]
It works on me without issue.

**Reply by z-huang** [reply]
Thanks for reporting! Now fixed.

**Comment by Randomboi06** [comment]
Can you make it for iphones pleaseeee i used this on my sister's phone and its crqzy, but when i searched it up i couldnt use it for iphone...

**Reply by Roxo7777** [reply]
It would be preferable for him to bring compatibility to iPhone after a really good stabilization, then yes, for now, it is immensely gratifying to have such an anticipated benefit 😈🫶🏻

**Comment by YellowRoseCx** [comment]
What are the new/changed Android Auto buttons? I also don't see the "Like" heart button anymore? 
<img src="https://github.com/user-attachments/assets/348d41bc-6ad3-499f-a9b4-ebbc783a6c7e" width=50% height=50%>

**Comment by aaru111** [comment]
why can't I restore my backup?

**Comment by Roxo7777** [comment]
On my brother's cell phone, a Honor X8b, despite having logged in to the web-view, the app won't

**Reply by Roxo7777** [reply]
(Android)

**Comment by HehehBoiii78** [comment]
There's one small issue I can mention here that has been lingering: If you go to Settings > Privacy > Clear listen/search history

It shows a dialog that reads "Are you sure to clear all listen/search history?" When it actually should be "Are you sure **_you want_** to clear all listen/search history?"

**Reply by z-huang** [reply]
Thanks!

**Comment by Snapxxx12** [comment]
please add audio fade in & fade out🙏

**Comment by Maxelite-895** [comment]
I download all my songs, but I noticed they weren't downloaded, I'm currently redownload them, but for some songs it isn't working.

**Comment by aaru111** [comment]
> * Discord RPC support!
> * In-app update checker
> * Player text alignment customization
> * Exclude downloaded files in auto backup
> * Fix random crash
> * Other fixes and enhancements
> 
> Screenshot
> _This discussion was created from the release [0.5.7](https://github.com/z-huang/InnerTune/releases/tag/v0.5.7)._

bro why cant i restore my data

**Reply by z-huang** [reply]
Probably because your made your backup in Malopieds' fork. Check out this tool: https://github.com/7heMech/InnerTune-Backup-Fix

**Reply by aaru111** [reply]
no it was your version 0.5.5 version

**Comment by Secretkillar** [comment]
**_This update is awesome. Please add listen together option for long distance 😊_**

**Comment by marcorizzo97** [comment]
Solving the problem of the app closing where the song playback player remains on the notification status bar

**Reply by Anonuser-code** [reply]
I got you. Just slide down the player bar in app. Done. It was hard for me to find the solution but here you go.

Same thing for youtube background videos if you use yt revanced or yt premium.

**Reply by Anonuser-code** [reply]
![IMG_20240820_234953](https://github.com/user-attachments/assets/974428b8-13bc-468a-8c3a-a64eb8d30ca3)
